docs/examples/rrv.ipynb

```diff
@@ -68,17 +66,16 @@
    ],
    "source": [
     "# Get data\n",
-    "data = pd.read_csv(\"https://raw.githubusercontent.com/neuropsychology/NeuroKit/dev/data/bio_eventrelated_100hz.csv\")\n",
-    "\n",
-    "# Visualize respiration\n",
-    "nk.signal_plot(data[\"RSP\"]) # Visualize"
+    "data = pd.read_csv(\"https://raw.githubusercontent.com/neuropsychology/NeuroKit/master/data/example_bio_100hz.csv\")\n",
+    "rsp = data[\"RSP\"]\n",
+    "nk.signal_plot(rsp) # Visualize"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now that we have the data, we can run the processing routine using `rsp_process()` that will compute the respiration rate and extract the inhalation peaks. The function outputs two things, a dataframe and a dictionnary."
```

Comment:
Yeah I think you didn't update the latest version in which I simplified this example. Could you restore it before we merge (https://github.com/neuropsychology/NeuroKit/blob/dev/docs/examples/rrv.ipynb)

Reply:
Done! Thanks for reviewing and commenting everything, really nice of you!